### PR TITLE
Fix execution scripts

### PR DESCRIPTION
**Pull request by dimkarakostas**

### CODE REVIEW

**Comment by dimkarakostas** [comment]
Fix #90 #104

File: backend/populate_victims.py [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+        tids.remove('')
+
+    try:
+        target_list = []
+        for t in tids:
+            target_list.append(Target.objects.get(id=int(t)))
+    except ObjectDoesNotExist:
+        print '[!] Invalid target id.'
+        exit(1)
+    return target_list
```
Comment (dionyziz) [inline backend/populate_victims.py]: DRY

File: backend/setup_backend.sh [hunk 2]
```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+BASEDIR=$(dirname "$0")
+
+echo '[-] Removing old database, if exists...'
+rm -f $BASEDIR/db.sqlite3
+if test -e "$BASEDIR/db.sqlite3"; then
+    echo '[!] Database could not be deleted';
```
Comment (dionyziz) [inline backend/setup_backend.sh]: "Database could not be deleted. Check permissions on $BASEDIR/db.sqlite3?"

File: backend/target_config.yml [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+    alignmentalphabet: 'ABCDEFGHIJKLMNOPQRSTUVWXYZ'
+    recordscardinality: 1
+
+dimkarakostas:
+    endpoint: 'https://dimkarakostas.com/rupture/test.php?ref=%s'
+    prefix: 'imper'
+    alphabet: 'abcdefghijklmnopqrstuvwxyz'
+    secretlength: 9
+    alignmentalphabet: 'ABCDEFGHIJKLMNOPQRSTUVWXYZ'
+    recordscardinality: 1
```
Comment (dionyziz) [inline backend/target_config.yml]: Nice.

**Comment by dionyziz** [comment]
LGTM, fix small comments I made and merge yourself.

File: realtime/deploy_realtime.sh [hunk 4]
```diff
@@ -1,7 +1,9 @@
...
 
-mkdir -p logs
+BASEDIR=$(dirname "$0")
+
+mkdir -p $BASEDIR/logs
 log_file=$(date "+%Y_%m_%d-%H_%M_%S")
 
 echo '[*] Realtime has been deployed.'
-npm start 2>&1 | tee logs/$log_file.log
+cd $BASEDIR && npm start 2>&1 | tee logs/$log_file.log
```
Comment (dionyziz) [inline realtime/deploy_realtime.sh]: Better than having --harmony flags in these scripts. Nice!

**Pull request merged**